### PR TITLE
Release version v0.19.0

### PR DESCRIPTION
**Pull request by josvazg**
Signed-off-by: Jose Luis Vazquez Gonzalez <josvaz@vmware.com>

**Description of the change**

Release version v0.19.0 release notes.

First release using the new release train process.

### CODE REVIEW

**Comment by josvazg** [comment]
This must be re-based after #1004 is merged

**Comment by josvazg** [comment]
As usual, used as a guide:
https://github.com/bitnami-labs/sealed-secrets/pulls?q=is%3Apr+is%3Aclosed+is%3Amerged

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ Latest release:
 
 [![](https://img.shields.io/github/release/bitnami-labs/sealed-secrets.svg)](https://github.com/bitnami-labs/sealed-secrets/releases/latest)
 
+## v0.19.0
+
+### Changelog
+
+- Support to recreate a deleted secret generated by the controller([#963](https://github.com/bitnami-labs/sealed-secrets/pull/963)).
```
Comment (agarcia-oss) [inline RELEASE-NOTES.md:11]:
```suggestion
- FEATURE: Support to recreate a deleted secret generated by the controller([#963](https://github.com/bitnami-labs/sealed-secrets/pull/963)).
```

**Review by agarcia-oss — APPROVED** [review]

**Review by alvneiayu — DISMISSED** [review]
LGTM

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ Latest release:
 
 [![](https://img.shields.io/github/release/bitnami-labs/sealed-secrets.svg)](https://github.com/bitnami-labs/sealed-secrets/releases/latest)
 
+## v0.19.0
+
+### Changelog
+
+- FEATURE: Support to recreate a deleted secret generated by the controller([#963](https://github.com/bitnami-labs/sealed-secrets/pull/963)).
+- Update golang.org/x/text ([#1008](https://github.com/bitnami-labs/sealed-secrets/pull/1008)).
```
Comment (alemorcuq) [inline RELEASE-NOTES.md:12]:
```suggestion
- Update `golang.org/x/text` fixing CVE-2022-32149 ([#1008](https://github.com/bitnami-labs/sealed-secrets/pull/1008)).
```

**Review by alemorcuq — CHANGES_REQUESTED** [review]

**Review by alemorcuq — APPROVED** [review]

**Pull request merged**